packages/aic/src/routes.ts

```diff
@@ -0,0 +1,164 @@
+import express from 'express'
+import { tid } from './tid'
+import { updateTick } from './document'
+import { sign } from './signature'
+import * as crypto from '@adxp/crypto'
+import { z } from 'zod'
+
+// Note: do not use the dev/test key in production
+//       pull the prod key from the secret store
+export const CONSORTIUM_KEYPAIR = crypto.EcdsaKeypair.import(
+  {
+    // did:key:zDnaeeL44gSLMViH9khhTbngNd9r72MhUPo4WKPeSfB8xiDTh
+    key_ops: [ 'sign' ],
+    ext: true,
+    kty: 'EC',
+    x: 'zn_OWx4zJM5zy8E_WUAJH9OS75K5t6q74D7lMf7AmnQ',
+    y: 'trzc_f9i_nOuYRCLMyXxBcpc3OVlylmxdESQ0zdKHeQ',
+    crv: 'P-256',
+    d: 'Ii__doqqQ5YYZLfKh-LSh1Vm6AqCWHGMrBTDYKaEWfU',
+  },
+  {
+    exportable: true,
+  },
+)
+
+let consortiumCrypto = async () => {
+    let key = (await CONSORTIUM_KEYPAIR)
+    return {
+    did: (): string => { return key.did() },
+    sign: async (msg:Uint8Array): Promise<Uint8Array> => {
+      return await key.sign(msg)
+    },
+    verifyDidSig: crypto.verifyDidSig,
+  }
+}
+// const literalSchema = z.union([z.string(), z.number(), z.boolean(), z.null()])
+// type Literal = z.infer<typeof literalSchema>
+// type Json = Literal | { [key: string]: Json } | Json[]
+// const jsonSchema: z.ZodType<Json> = z.lazy(() =>
+//   z.union([literalSchema, z.array(jsonSchema), z.record(jsonSchema)]),
+// )
+
+// const Patch = z.tuple([
+//   z.literal('put').or(z.literal('del')),
+//   z.string().array(),
+//   jsonSchema,
+// ])
+// type Patch = z.infer<typeof Patch>
+
+// const Diff = z.object({
+//   prev: z.string().regex(/[2-7a-z]{4}-[2-7a-z]{3}-[2-7a-z]{4}-[2-7a-z]{2}/),
+//   patches: Patch.array(),
+//   // base58btc alphabet 123456789 ABCDEFGH JKLMN PQRSTUVWXYZ abcdefghijk mnopqrstuvwxyz
+//   key: z.string().regex(/did:key:z[1-9A-HJ-NP-Za-km-z]+/),
+//   sig: z.string().regex(/z[1-9A-HJ-NP-Za-km-z]+/),
+// })
+// type Diff = z.infer<typeof Diff>
+
+const router = express.Router()
+router.use('/favicon.ico', express.static('static/favicon.ico'))
+
+router.all('/tid', async (req, res) => {
+  res.send(
+    await sign(
+      {
+        tid: tid(),
+        key: (await consortiumCrypto()).did(),
+        sig: '',
+      },
+      await consortiumCrypto(),
+    ),
+  )
+})
+
+router.get(
+  '/:pid([234567abcdefghijklmnopqrstuvwxyz]{16})/',
+  async function (req, res) {
+    // AIC git
+    //
+    // Retreve the latest tick from the databace
+    // pass the tick and curent tid to the aic lib for signing
+    const did = 'did:aic:' + req.params.pid
+    const row = await res.locals.db.tickForDid(did) // retreve latest tick
+    const prev_tick = row ? JSON.parse(row.tick) : null
+    const signedDoc = await updateTick(
+      did,
+      tid(),
+      null, // candidateDiff: this is a get no updates
+      prev_tick,
+      await consortiumCrypto(),
+    )
+    res.type('json').send(signedDoc)
```

Comment:
We need to send a proper status code if there's an error. Even just a 500. But right now we're sending a `200` with an error msg